openlibrary/core/helpers.py

```diff
@@ -269,6 +270,14 @@ def affiliate_id(affiliate):
 def bookreader_host():
     return config.get('bookreader_host', '')
 
+def private_collections():
+    """Collections which are lendable but should not be linked from OL
+    TODO: Remove when we can handle institutional books"""
+    return ['georgetown-university-law-library-rr']
```

Comment:
This may be a one time thing. (Ha!) But sure, let's discuss alternate ways of configuring this.